README.md

```diff
@@ -23,28 +23,73 @@ Currently Supporting:
 - bower
 - ios-webkit-debug-proxy
 - ideviceinstaller
+- java
+- ant
 
 See the [validatorRules.js file][validator] file for the full list of things that are supported.
 
-### Usage
+## Usage
+
+### CLI
+
+#### Install
+Can be installed globally or in a local directory.
+
+- **Globally**: `npm install -g https://github.com/mohlsen/check-engine.js.git`
+- **Local**: `npm install https://github.com/mohlsen/check-engine.js.git`
+
+#### Running:
+1. Run `check-engine [path_to_package.json]`. `check-engine` located in `/bin/check-engine` if using a local install.
```

Comment:
I think "...`check-engine` is located in `/bin/check-engine`..." Sounds a little better.